src/test/kotlin/no/nav/familie/oppdrag/simulering/mock/readme.md

```diff
@@ -0,0 +1,68 @@
+# Dokumentasjon PeriodeGenerator
+Oppdraget som kommer inn via Simulering WS inneholder oppdragslinjer av typen OPPH eller YTEL.
+Først leses alle disse oppdragslinjene gjennom for fordeler OPPH og YTEL linjer på hver sin liste
```

Comment:
Skulle for vært og? "gjennom **og** fordeler"

Reply:
endret til gjennom og fordeler